controllers/snapshot/snapshot_adapter_test.go

```diff
@@ -71,9 +70,13 @@ var _ = Describe("Snapshot Adapter", Ordered, func() {
 				Bundle:      "quay.io/kpavic/test-bundle:component-pipeline-pass",
 				Pipeline:    "component-pipeline-pass",
 				Environment: integrationv1alpha1.TestEnvironment{
-					Name:   "envname",
-					Type:   "POC",
-					Params: []string{},
+					Name: "envname",
+					Type: "POC",
+					Configuration: applicationapiv1alpha1.EnvironmentConfiguration{
+						Env: []applicationapiv1alpha1.EnvVarPair{
+							{Name: "", Value: ""},
```

Comment:
Is this line needed? Haven't tried myself, but it seems you are just setting nil values.

Reply:
Updated them to some temp values and will use them in the coming test case later. Thanks